hashicorp/boundary-ui
# Llb/worker filter improvements

# Description

This is all of the worker filter improvement changes ready to be merged into main. All of this work has been approved in smaller PRs into the llb but we wanted to give the whole team time to review these changes.




## How to Test

The changes updated how we add or edit worker filters for Targets, Vault Credential Stores, and Storage Buckets.
You should also see a "Filter Generator" now as well. It should be hidden by default and has a toggle to show it if needed. The generator does not persist any data and should be empty any time it is shown.

## Checklist
